static/login.html

```diff
@@ -14,10 +14,12 @@
   </head>
   <body>
     <img id="wordmark" src="../images/moziot-wordmark.png" />
-    <form id="login-form" class="user-form" action="/login" method="post">
-      <input type="email" name="email" placeholder="Email"></input>
-      <input type="password" name="password" placeholder="Password"></input>
+    <form id="login-form" class="user-form" method="post">
+      <input type="email" id="email" name="email" placeholder="Email"></input>
+      <input type="password" id="password" name="password" placeholder="Password"></input>
       <button id="login-button" type="submit">Log in</button>
     </form>
+    <script type="text/javascript" src="js/api.js"></script>
+    <script type="text/javascript" src="js/login.js"></script>
```

Comment:
Currently, this results in there being no way to log back in after logging out the initially created user